src/cascadia/TerminalControl/ControlInteractivity.cpp

```diff
@@ -340,7 +340,7 @@ namespace winrt::Microsoft::Terminal::Control::implementation
                                             const Core::Point pixelPosition,
                                             const bool pointerPressedInBounds)
     {
-        const auto terminalPosition = _getTerminalPosition(til::point{ pixelPosition }, true);
+        const auto terminalPosition = _getTerminalPosition(til::point{ pixelPosition }, false);

```

Comment:
Ah, I see why this is confusing. I was wondering how this change _didn't_ impact selection. This variable is defined early and used late.... and selection ignores it. Wow.